nmdc_runtime/site/ops.py

```diff
@@ -847,3 +905,45 @@ def get_csv_rows_from_url(url: Optional[str]) -> List[Dict]:
     # Collect all the rows into a list of dicts while stripping out (valfilter) cells where the
     # value is an empty string (identity returns a Falsy value).
     return [valfilter(identity, row) for row in reader]
+
+
+@op
+def get_df_from_url(url: str, delimiter: str = ",") -> pd.DataFrame:
+    """Download and return a pandas DataFrame from a given URL.
+
+    :param url: raw URL of the file to be downloaded as a DataFrame
+    :param delimiter: delimiter used to separate rows/columns in the file, defaults to ","
+    :return: pandas DataFrame of CSV/TSV/etc content
+    """
+    if not url:
+        return pd.DataFrame()
```

Comment:
this seems an odd pattern. Better to make teh signature include `Optional` and return `None`?